elements/pfe-primary-detail/src/pfe-primary-detail.js

```diff
@@ -88,6 +100,7 @@ class PfePrimaryDetail extends PFElement {
 
   constructor() {
     super(PfePrimaryDetail, { type: PfePrimaryDetail.PfeType });
+    this.isIE = !!window.MSInputMethodContext && !!document.documentMode;
```

Comment:
+1 This is a great approach ^

Reply:
Didn't even notice I had that other code in there still 😂 
Gone now